passport/src/auth-handler.ts

```diff
@@ -164,7 +166,7 @@ function createPerDomainAuth(domain: string, strategy: passport.Strategy) {
     );
   }
   router.all('/logged-in', (_req, res) => {
-    return res.header('content-type', 'text/html')
+    res.header('content-type', 'text/html')
```

Comment:
Why did we have to get rid of `return` here?  React?

Reply:
This is the same rule as the change in https://github.com/reshufflehq/reshuffle/pull/358/files#diff-3e34d3ded243ab6635707c51d43e10efR177 -- [arrow-body-style](https://eslint.org/docs/rules/arrow-body-style).

Not having a `return` allows to keep braces and indentation unchanged - it behaves slightly different than tslint `arrow-return-shorthand`.